src/components/home/directory/viewer/dataModel/DataModel.vue

```diff
@@ -52,35 +52,35 @@ async function addPropertiesAndTypes(iri: any) {
 
   result.forEach((r: any) => {
     properties.push({
-                      id: r.property["@id"],
-                      parents: [iri],
-                      name: r.property.name || r.property["@id"],
-                      type: "property",
-                      cardinality: `${r.minExclusive || r.minInclusive || 0} : ${r.maxExclusive || r.maxInclusive || "*"}`
-                    });
+      id: r.property["@id"],
+      parents: [iri],
+      name: r.property.name || r.property["@id"],
+      type: "property",
+      cardinality: `${r.minExclusive || r.minInclusive || 0} : ${r.maxExclusive || r.maxInclusive || "*"}`
```

Comment:
consider extracting this functionality into ha library helper, as cardinality is calculated in multiple components

Reply:
will be moving logic to backend